DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/EvaluationSummaryViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models.TrackingSystem;
+
+    public class EvaluationSummaryViewModel
+    {
+        public EvaluationSummaryViewModel(EvaluationSummaryModel model)
+        {
+            Question = model.Question;
+            if (model.ResponseCounts != null)
+            {
+                ResponsePercentages = model.ResponseCounts
+                    .Select(x => (question: x.response, percentage: 100f * x.count / model.TotalResponses));
+            }
+        }
+
+        public string Question { get; set; }
+
+        public IEnumerable<(string response, float percentage)>? ResponsePercentages { get; set; }

```

Comment:
Do you think it'd be worth setting this as non-nullable (and default to empty array), then we don't need to handle empty array AND null value? I'm assuming both would be equivalent in front end.

Reply:
I feel like it's clearer if null represents the fact that there is no data?

Reply:
That's fair - can we add the handling the empty array case in frontend though?